pkg/chart/fake/chart.go

```diff
@@ -33,18 +35,40 @@ func (f *FakeChart) ParseDetails(data []byte) (*chartUtils.Details, error) {
 }
 
 func (f *FakeChart) GetChart(details *chartUtils.Details, netClient chartUtils.HTTPClient, requireV1Support bool) (*chartUtils.ChartMultiVersion, error) {
+	vals, err := getValues([]byte(details.Values))
+	if err != nil {
+		return nil, err
+	}
 	return &chartUtils.ChartMultiVersion{
-		Helm2Chart: &chart.Chart{
-			Metadata: &chart.Metadata{
+		Helm2Chart: &chart2.Chart{
+			Metadata: &chart2.Metadata{
 				Name: details.ChartName,
 			},
-			Values: &chart.Config{
+			Values: &chart2.Config{
 				Raw: details.Values,
 			},
 		},
+		Helm3Chart: &chart3.Chart{
+			Metadata: &chart3.Metadata{
+				Name: details.ChartName,
+			},
+			Values: vals,
+		},
 	}, nil
 }
 
+// Values is a type alias for values.yaml
+type Values map[string]interface{}
```

Comment:
Not sure that there's any benefit hiding the `map[string]interface{}` in a type alias?

Reply:
I thought it would make it easier to read function signatures.
_removed in latest commit_